src/pages/landing/[landingPage]/[article]/index.tsx

```diff
@@ -10,10 +13,31 @@ import { SingleArticle } from 'components/SingleArticle/SingleArticle'
 const LandingPageArticle = ({
   article,
 }: InferGetServerSidePropsType<typeof getServerSideProps>) => {
+  const [breadcrumbNavItems] = useState(window.location.pathname.split('/'))
+
   return (
-    <GridContainer>
-      <SingleArticle article={article} />
-    </GridContainer>
+    <>
+      <PageHeader searchDisplay={false}>
+        <BreadcrumbNav
+          navItems={[
+            { path: '/', label: 'Service portal home' },
+            { path: '/landing', label: 'Landing pages' },
+            {
+              path: `/landing/${breadcrumbNavItems[2]}`,
+              label: `${breadcrumbNavItems[2].toUpperCase()}`,
+            },
+            {
+              path: `/landing/${breadcrumbNavItems[2]}/${breadcrumbNavItems[3]}`,
+              label: `${breadcrumbNavItems[3]}`,
+              current: true,
```

Comment:
I thought it would be fine to hardcode the array positions here for now, but we could write some logic to make it a bit more dynamic in the future if need be.